packages/cli/src/helpers/getUnappliedSchemaOverridesTableNames.ts

```diff
@@ -0,0 +1,29 @@
+import { isObject } from './isValidSchemaDefOverrides';
+
+export function getUnappliedSchemaOverridesTableNames(
+    tables: string[],
+    schemaDefOverrides: unknown,
```

Comment:
You shuold first validate this type, using the JSON schema approach we mentioned earlier and then pass the right type here.